### ISSUE TITLE
[Minor] Rename the default Conda environment from `workflow` to `base`

### ISSUE THREAD

**Issue by peterschmidt85** [dstackai/dstack#1385]

**Comment by peterschmidt85** [comment]
Relates to #1379

**Comment by abubakarp789** [comment]
Assign me this issue to resolve

**Comment by JanumalaAkhilendra** [comment]
If this Issue is still open , I would like to work on this, please assign it to me .
Thank you.

**Comment by fool1280** [comment]
Hi, may I ask whether this is a good idea? Because I believe conda also has its own base environment?

**Comment by peterschmidt85** [comment]
@fool1280 True. If there is no way to replace the default Conda environment with ours (can it be done?), then we need to think of an alternative plan.

**Comment by anhkumo** [comment]
What were the motivations for this change in the beginning? 

Personally, I feel "workflow" is easy to understand than "base".

**Comment by fool1280** [comment]
@peterschmidt85 What were the motivations for this change?

Personally, I feel "workflow" is easier to understand than "base".

**Comment by peterschmidt85** [comment]
@fool1280 workflow is a legacy name. previously we called runs - workflows. now we don't use the term workflow anymore. perhaps anything else more neutral would just do. 
again - it's not a big deal - its just cosmetics

**Comment by r4victor** [comment]
No longer relevant after #2652
